<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AfficheActionnaire</class>
 <widget class="QDialog" name="AfficheActionnaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QLabel" name="TextIdAc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Id Actionnaire</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValIdActionnaire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextNomAc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom Actionnaire</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValNomActionnaire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextMntAc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Montant Emis</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValMntActionnaire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextTDAc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Taux dividende</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValTDActionnaire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextDAc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dividende</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValDActionnaire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextDAAc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date d'achat</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValDAActionnaire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValStatutActionnaire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TextStatutAc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Statut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnregistrerActionnaire">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
  <widget class="QLabel" name="TextDureeAc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dur√©e</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValDureeActionnaire">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
